Lib/pathlib/_abc.py

```diff
@@ -205,8 +205,8 @@ class PurePathBase:
     )
     pathmod = posixpath
 
-    def __init__(self, *paths):
-        self._raw_paths = paths
+    def __init__(self, *args):
+        self._raw_paths = list(args)
```

Comment:
Why should it be a list?

Reply:
`PurePath._raw_paths` is a list. `PurePathBase` has only just been split off from `PurePath`, and in doing so, I mistakenly changed the type of `_raw_paths` to a tuple, but only in `PurePathBase` and not `PurePath`. So the answer is: for consistency with its subclass, and to correct a mistake.

You could reasonably ask "why is `PurePath._raw_paths` a list?". The answer: `PurePath.__init__()` builds up the path list incrementally, and performs some special handling of other `PurePath` objects passed in as arguments. It's cheaper to use the list produced there as-is than convert it to a tuple/similar.

Reply:
Perhaps I should add a comment here to explain why we convert it to a list?